# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnuchess/gcc4.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur gnuchess-5.07-orig/src/input.c gnuchess-5.07/src/input.c
--- gnuchess-5.07-orig/src/input.c	2003-07-01 17:06:43.000000000 +0000
+++ gnuchess-5.07/src/input.c	2006-01-25 19:47:32.683779750 +0000
@@ -92,7 +92,7 @@
   fgets(inputstr, MAXSTR, stdin);
 }
 
-static pthread_t input_thread;
+pthread_t input_thread;
 
 /* Mutex and condition variable for thread communication */
 
